nni/compression/pytorch/compressor.py

```diff
@@ -589,8 +589,26 @@ class QuantGrad(torch.autograd.Function):
     """
     Base class for overriding backward function of quantization operation.
     """
+    @classmethod
+    def _quantize(cls, x, scale, zp):
+        r"""Reference function for quantizing x -- non-clamped.
+        """
+        return ((x / scale) + zp).round()
+    @classmethod
+    def get_bits_length(cls, config, quant_type):
+        """
+        get bit for quantize config
+        :param config:
+        :param quant_type:
+        :return:
```

Comment:
please update docstring to be consistent with others

Reply:
cool